scripts/update-to-latest-repo.sh

```diff
@@ -1,29 +1,71 @@
 #!/bin/bash
-INSTALL_PATH="./backend.ai-dev"
-BRANCH_VERSION="main"
+
+shopt -s xpg_echo
+
+RED="\033[0;91m"
+GREEN="\033[0;92m"
+YELLOW="\033[0;93m"
+BLUE="\033[0;94m"
+CYAN="\033[0;96m"
+WHITE="\033[0;97m"
+LRED="\033[1;31m"
+LGREEN="\033[1;32m"
+LYELLOW="\033[1;33m"
+LBLUE="\033[1;34m"
+LCYAN="\033[1;36m"
+LWHITE="\033[1;37m"
+LG="\033[0;37m"
+NC="\033[0m"
+REWRITELN="\033[A\r\033[K"
+
+echo "${LGREEN}Backend-ai: Update-to-latest-repo ${NC}"
+
+read -e -p "Enter Your BRANCH_VERSION:" -i "main" BRANCH_VERSION
+echo "${YELLOW}${BRANCH_VERSION}${NC}"
+
+read -e -p "Enter Your INSTALL_PATH:" -i "./backend.ai-dev" INSTALL_PATH
+echo "${YELLOW}${INSTALL_PATH}${NC}"
```

Comment:
Please allow non-interactive usage by accepting these values from the arguments or environment variables. [Consult the bash parameter substitution docs.](https://tldp.org/LDP/abs/html/parameter-substitution.html)